perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -54,6 +135,128 @@
                   'num_machines': 1}
 
 
+GLOBALS_TEMPLATE = """
+[global]
+ioengine=libaio
+invalidate=1
+direct=1
+runtime=10m
+time_based
+filename={{filename}}
+do_verify=0
+verify_fatal=0
+randrepeat=0
+"""
+
+JOB_TEMPLATE = """
+{% for iodepth in iodepths %}
+[{{name}}-io-depth-{{iodepth}}]
+stonewall
+rw={{rwkind}}
+blocksize={{blocksize}}
+iodepth={{iodepth}}
+size={{size}}
+{% endfor %}
+"""
+
+
+def GetIODepths(io_depths):
+  """Parse the io_depths parameter.
+
+  Args:
+    io_depths: a string in the format of the --io_depths flag.
+
+  Returns:
+    An iterable of integers.
+
+  Raises:
+    ValueError if io_depths doesn't follow a format it recognizes.
+  """
+
+  match = IODEPTHS_REGEXP.match(io_depths)
+
+  if match.group(2) is None:
+    return [int(match.group(1))]
+  else:
+    return range(int(match.group(1)), int(match.group(3)) + 1)
+
+
+def GenerateJobFileString(disk, against_device,
+                          scenarios, io_depths, working_set_size):
+  """Write a fio job file.
```

Comment:
The documentation for this method is not accurate. This method does not write a job file, it just generate the job file string.